cloud: added integrations (#10924)
pingcap/docs: tidb-cloud/third-party-integrations.md

---

> title: Third-party Integrations

Reviewer: ```suggestion
title: Third-party Monitoring Integrations
```

Reviewer 2: +1

> summary: Learn how to use third-party integrations.

Reviewer: ```suggestion
summary: Learn how to use third-party monitoring integrations.
```

---

> # Third-Party Integrations

Reviewer: ```suggestion
# Third-Party Monitoring Integrations
```

> 

Reviewer: ```suggestion

You can integrate TiDB Cloud with third-party monitoring services to receive TiDB Cloud alerts and view the performance metrics of your TiDB cluster using the monitoring services.

```

> ## Required access

Reviewer: ```suggestion

> **Note:**
>
> For [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier), third-party monitoring integrations are not supported.
 
## Required access
```

> To edit third-party integration settings, you must have the `Organization Owner` access to your organization or `Project Member` access to the target project.

Reviewer: ```suggestion
To edit third-party integration settings, you must have the `Owner` access to your organization or `Member` access to the target project.
```

## View or modify third-party integrations

1. On the TiDB Cloud console, choose a target project that you want to view or modify, and then click the **Project Settings** tab.
2. In the left pane, click **Integrations**. The available third-party integrations are displayed.

## Available integrations

### Datadog integration

> Configures TiDB Cloud to send metric data about your TiDB clusters to [Datadog](https://www.datadoghq.com/). You can view these metrics in your Datadog dashboards. To get a detailed list of all metrics that Datadog tracks, refer to [Datadog Integration](/tidb-cloud/monitor-datadog-integration.md).

Reviewer: ```suggestion
With the Datadog integration, you can configure TiDB Cloud to send metric data about your TiDB clusters to [Datadog](https://www.datadoghq.com/) and view these metrics in your Datadog dashboards.

For the detailed integration steps and a list of metrics that Datadog tracks, refer to [Integrate TiDB Cloud with Datadog](/tidb-cloud/monitor-datadog-integration.md).
```

### Prometheus and Grafana integration

> Get a scrape_config file for Prometheus from TiDB Cloud and use the content from the file to configure Prometheus. You can view these metrics in your Grafana dashboards. To get a detailed list of all metrics that Prometheus tracks, refer to [Prometheus and Grafana Integration](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).

Reviewer: ```suggestion
With the Prometheus and Grafana integration, you can get a scrape_config file for Prometheus from TiDB Cloud and use the content from the file to configure Prometheus. You can view these metrics in your Grafana dashboards. 

For the detailed integration steps and a list of metrics that Prometheus tracks, see [Integrate TiDB Cloud with Prometheus and Grafana](/tidb-cloud/monitor-prometheus-and-grafana-integration.md).
```

> ---

Reviewer: Would you please add "monitoring" to the filename? Thanks

Author: Done
